src/modules/header/desktopNavigation/desktopNavigation.js

```diff
@@ -24,6 +24,9 @@ export class BaseDesktopNavigation extends React.Component {
       },
       girls: {
         visible: false
+      },
+      baby: {
```

Comment:
What do you think about using a single state variable to keep track of which mega menu to display? I think we don't want to ever show two mega menus at the same time, but that would be possible with the current setup, e.g. we have `state.girls.visible = true` and `state.baby.visible = true`. 

I think it might be simpler to just have a single variable on state to track this, maybe something like: `visibleMenu`, it would be a `String` which values could be: `null`, `'boys'`, `'girls'` or `'baby'`.

Reply:
I just tried implementing this but there's quite a bit of logic dependent on the way that we're handling the visible state ☹️ We're currently rendering all three of the `MegaMenus` at the same time regardless of whether they're visible and perform a fadein/fadeout animation as we toggle between them. As it's written, there doesn't seem to be a circumstance that would cause multiple menus to be shown at the same time, even though the data structure of our state seems to suggest it.

I think it'd be a good candidate for refactoring in the future!